azure-pipelines.yml

```diff
@@ -0,0 +1,43 @@
+resources:
+  containers:
+    - container: node10
+      image: 10.15.3-alpine
+    - container: node11
+      image: 11.12.0-alpine
+    - container: python3
+      image: 3.7.2-alpine
+jobs:
+  - job: root
+    strategy:
+      matrix:
+        node10:
+          container: node10
+        node11:
+          container: node11
+    pool:
+      vmImage: ubuntu-16.04
+    container: $[variables[container]]
+    steps:
+      - checkout: self
+      - script: yarn --no-progress install
+      - script: yarn --silent prettylint --ignore-path .gitignore '**/*.{json,md,yml}' '!packages/**'
```

Comment:
Should this script be added to the root `package.json`?

Reply:
I don't think this is something we will need to run from the command line. I'm more of the philosophy that we can move it if/when we have a reason to. Otherwise, this serves the purpose here.

Reply:
I can see us running it from the command line, but the reason I was contemplating whether or not to add it to the root `package.json` is that Husky will run `prettylint` against files with those extensions when they are staged. So it was more, should we add a way to address those potential issues ourselves prior to pre-commit, or just wait for Husky to catch them during the pre-commit process.

Reply:
I think this is okay as-is. If we have a need for it later we can move it.